src/models/client.ts

```diff
@@ -183,12 +189,17 @@ export class Client extends EventEmitter {
           }
         : options.clientProperties
 
+    if (options.shard !== undefined) this.shard = options.shard
+    if (options.shardCount !== undefined) this.shardCount = options.shardCount
+
     this.slash = new SlashClient({
       id: () => this.getEstimatedID(),
       client: this,
       enabled: options.enableSlash
     })
 
+    if (options.fetchGatewayInfo === true) this.fetchGatewayInfo = true
```

Comment:
I think it's better to just do like `this.fetchGatewayInfo = options.fetchGatewayInfo ?? false`

Reply:
Oh.. yes